unique_toolkit/tests/_common/token/test_token_counting.py

```diff
@@ -0,0 +1,289 @@
+# Original source
+# https://github.com/openai/openai-cookbook/blob/main/examples/How_to_count_tokens_with_tiktoken.ipynb
+
+from enum import StrEnum
+from pathlib import Path
+
+import tiktoken
+from PIL import Image
+
+from unique_toolkit._common.token.token_counting import (
+    SpecialToolCallingTokens,
+    num_tokens_for_tools,
+    num_tokens_from_messages,
+)
+from unique_toolkit._common.utils.image.encode import image_to_base64
+
+CURRENT_DIR = Path(__file__).parent.absolute()
+
+
+class OpenAIModelName(StrEnum):
```

Comment:
We have a list of all models with names in the toolkit. Wouldn't it make sense to reuse that enum instead of creating this here?